### PR TITLE
Split Names in Volunteer Sign In 

### PR DESCRIPTION
**Pull request by DataFighter**
Split Names in Volunteer Sign up
Originally, at sign in, volunteers typed in their full name. 
We are now changing it so that volunteers can type in each name individually.

### CODE REVIEW

File: uber/models.py [hunk 1]
```diff
@@ -492,13 +492,10 @@ def get_account_by_email(self, email):
...
             return not self.query(Email).filter_by(subject=subject).all()
 
-        def lookup_attendee(self, full_name, email, zip_code):
-            words = full_name.split()
-            for i in range(1, len(words)):
-                first, last = ' '.join(words[:i]), ' '.join(words[i:])
-                attendee = self.query(Attendee).iexact(first_name=first, last_name=last, email=email, zip_code=zip_code).all()
-                if attendee:
-                    return attendee[0]
+        def lookup_attendee(self, input_first_name, input_last_name, email, zip_code):
```
Comment (EliAndrewC) [inline uber/models.py]: Looks good, but can you rename these parameters from ``input_first_name`` and ``input_last_name`` to just be ``first_name`` and ``last_name``?  (I'm kind of nitpicky about variable names.)

**Comment by EliAndrewC** [comment]
Looks great!  I have a request for a variable name change, but after that this should be good to merge - thanks!

File: uber/models.py (hunk 1 above)
Comment (DataFighter) [inline uber/models.py]: ok rgr

**Comment by EliAndrewC** [comment]
+1

File: uber/models.py (hunk 1 above)
Comment (kitsuta) [inline uber/models.py]: Whoops! It looks like you renamed the variables in the line below, but not in the function definition. Eli didn't catch this but can you fix it real quick and submit another PR? It won't work right without that fix.

Comment (EliAndrewC) [inline uber/models.py]: Good catch!  Also please make sure to re-test before making the second PR to make sure we didn't accidentally break anything again.  Thanks!
